ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install.sh

```diff
@@ -58,6 +58,11 @@ cp $UBUNTU_COMMON_DIR/check_nvidia_uvm.sh .
 $UBUNTU_COMMON_DIR/check_uvm_service.sh
 
 
+#rename IPoIB interfaces
+ibps="\nACTION==\"add\", SUBSYSTEM==\"net\", DRIVERS==\"?*\", ATTR{type}"
+ibps+="==\"32\", KERNEL==\"ib*\", NAME=\"ib0\""
+echo -e $ibps | sudo tee -a /etc/udev/rules.d/70-persistent-ipoib.rules
+
```

Comment:
Could you please describe the naming issue of IPoIB device in Ubuntu20.04, and why we need it? 
Also is it ensured that the IPoIB device can be precisely matched with the following rule:
`ACTION=="add", SUBSYSTEM=="net", DRIVERS=="?*", ATTR{type}=="32", KERNEL=="ib*", NAME="ib0"`?

I'd be better if we add these info into the commit msg.

Reply:
Thanks for your clarification, @moes1. 
Are we doing this renaming to unify the IPoIB interface name across Ubuntu 1804 and 2004, or the naming in ubuntu2004 (like ibP257s154062) breaks something?

Reply:
I was doing the renaming only in ubuntu2004 just so the naming scheme was the same in both 1804 and 2004.